docs/source/notebooks/api_quickstart.ipynb

```diff
@@ -16,12 +16,13 @@
      "name": "stdout",
```

Comment:
<p>Can you add a few lines about what <code>return_inferencedata</code> does, to introduce the difference with the default behavior, and the advantages of using <code>InferenceData</code>? You can link to <a href="https://arviz-devs.github.io/arviz/notebooks/Introduction.html" target="_blank">ArviZ quickstart</a> at the end.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>